### PR TITLE
Update coordinatePoint.schema.json

### PR DESCRIPTION
**Pull request by UlrikeS91**
- update `$id`
- update `@id` and `@type` #48/#49
- update `coordinatePoint` description

### CODE REVIEW

File: v0.1/elements/coordinatePoint.schema.json [hunk 1]
```diff
@@ -14,12 +14,11 @@
     "@type": {
       "type": "string",
       "description": "Metadata node type (schema).",
-      "const": "https://schema.ebrains.eu/sands/CoordinatePoint",
+      "const": "https://openminds.ebrains.eu/sands/coordinatePoint",
```
Comment (skoehnen) [inline v0.1/elements/coordinatePoint.schema.json:17]: One comma too much

**Comment by skoehnen** [comment]
Found an error.
I will fix it and start a new pull request.

**Comment by UlrikeS91** [comment]
I fixed it in the branch, you can also check it here and merge it now :)

**Comment by skoehnen** [comment]
Ah, I also add a commit to fix it :)

**Comment by skoehnen** [comment]
I will just merge yours, and delete my new pull request.

**Comment by UlrikeS91** [comment]
Either way, we can merge this now right?

**Comment by skoehnen** [comment]
> Either way, we can merge this now right?

Yes 😉

**Pull request merged**